Customizing/README.md

```diff
@@ -0,0 +1,68 @@
+# Customizing
+
+<!-- MarkdownTOC depth=0 autolink="true" bracket="round" autoanchor="true" style="ordered" indent="   " -->
+
+1. [Introduction](#introduction)
+1. [User Agreements](#user-agreements)
+1. [System Language Changes](#system-language-changes)
+1. [Skins and Styles](#skins-and-styles)
+
+<!-- /MarkdownTOC -->
+
+<a name="introduction"></a>
```

Comment:
AFAIK this is not necessary, GitHub-Markdown will include these automatically. Please cross check via [View].